scripts/download-vsix-from-circleci.js

```diff
@@ -66,6 +66,14 @@ if (
   process.exit(-1);
 }
 
+// Check release version environment variable
+if (!nextVersion.match(/^(\d+)\.(\d+)\.(\d+)$/)) {
```

Comment:
This validation is also in the `publish-circleci.js` script, that's because we can call both scripts independently.